.changelog/unreleased/features/2765-bls12-381-curve.md

```diff
@@ -0,0 +1,2 @@
+- `[crypto]` Add support for BLS12-381 keys. Use `bls12381` build flag to enable
```

Comment:
Could you insert this?:

_Since it needs `cgo`, we use `bls12381` build flag to enable..._

Reply:
We also use `bls12381` build flag because we don't want to bring in new dependencies if the user doesn't need bls.

Reply:
Agree with you